.github/workflows/ci.yml

```diff
@@ -172,3 +148,63 @@ jobs:
 
       - name: Run Plugin-Web tests
         run: yarn ci:test:plugin-web
+
+  test-playwright:
+    strategy:
```

Comment:
the github workflow json schema seems to require `matrix` as a property for `strategy`, and we are not defining a matrix here.

If we are not running this in a matrix I don't think we need the `strategy`

Reply:
ref: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategy

Reply:
thanks for pointing it out, removed here + in the web tests.